### ISSUE TITLE
Live-Reload multiple files

### ISSUE THREAD

**Issue by oschmid** [metasoarous/oz#186]
Is live-reload of multiple files possible? (In code or from a repl?) My project is growing too big for a single file...

**Comment by metasoarous** [comment]
Yes; Absolutely. You can either `live-view!` or `build!` a directory of input files.

**Comment by oschmid** [comment]
Thanks for the help! `live-view!` doesn't seem to accept a directory but `build!` will.

**Comment by metasoarous** [comment]
Got it! Thanks for catching this; It _should_, so I'm going to update the issue title.
